legion/operator/pkg/utils/ssh.go

```diff
@@ -13,7 +29,7 @@ const PublicSshKeyName = "ssh_known_hosts"
 
 var (
 	SshUrlRegexp = regexp.MustCompile(`(?P<User>\w+)@(?P<Host>[\w.]+):.*`)
```

Comment:
What about protocol prefixes? Like `git+ssh://user@host`

Reply:
Fixed and wrote unit tests